src/translate_c.zig

```diff
@@ -72,6 +72,12 @@ const Scope = struct {
         /// so that the return expression can be cast, if necessary
         return_type: ?clang.QualType = null,
 
+        /// C static local variables are wrapped in a block-local struct. The struct
+        /// is named after the (mangled) variable name, the Zig variable within the
+        /// struct itself is given this name. We use a dot in the name to ensure
+        /// it does not clash with any C identifiers
+        const StaticInnerName = ".inner";
```

Comment:
Could we use `static` instead? I think `foo.static` makes it clearer why the code looks the way it is and  using a C keyword also takes care of the shadowing issue.

Reply:
ah nice, good idea